torch/testing/_internal/common_utils.py

```diff
@@ -623,19 +635,19 @@ def run_tests(argv=UNITTEST_ARGS):
     # import test files.
     if IMPORT_SLOW_TESTS:
         if os.path.exists(IMPORT_SLOW_TESTS):
-            with open(IMPORT_SLOW_TESTS, 'r') as fp:
+            with open(SLOW_TESTS_FILE, 'r') as fp:
+                global slow_tests_dict
+                slow_tests_dict = json.load(fp)
                 # use env vars so pytest-xdist subprocesses can still access them
-                os.environ['SLOW_TESTS_DICT'] = fp.read()
+                os.environ['SLOW_TESTS_DICT'] = "1"
```

Comment:
Now that we are not going to store anything here and use this as a flag, may be it's a good idea to rename this to something clearer like `os.environ['USE_SLOW_TESTS'] = True`, thoughts?